gdal/port/cpl_string.cpp

```diff
@@ -2505,7 +2505,7 @@ char *CPLBinaryToHex( int nBytes, const GByte *pabyData )
  * @return hexadecimal translation, zero terminated.  Free with CPLFree().
  */
 
-char *CPLBEBinaryToHex( int nBytes, const GByte *pabyData )
+char *CPLBigEndianBinaryToHex( int nBytes, const GByte *pabyData )
```

Comment:
I believe we decided to remove that and use CPLBinaryToHex() instead. You'd just have to indicate in the doc that the ordering of number is little endian.

Reply:
Yes, you are right. I'm not using this `CPLBigEndianBinaryToHex` on the DGN driver. 

I thought it might be useful else where and that's why I've kept it on CPL. I'm still using it on the DGNV8 driver. On the DGNV8, I'm trying to provide the data interpreted, as possible. 

I can make it a local function on DGBV8 driver and remove it from CPL. Probably it does not make sense to have it in the CPL.

Reply:
I'll remove it from CPL.